### PR TITLE
Generate ruby client

### PR DESCRIPTION
**Pull request by nemanja-m**

### CODE REVIEW

File: lib/mix/tasks/ruby.ex [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    Mix.shell.info "\n-- Generating ruby client\n"
+
+    generate_folder_structure
+    generate_gemspec
+    generate_gemfile
+    generate_lib
+    generate_bin
+    generate_thrift_files
+
+    Mix.shell.info "-- \e[32mDone!\n"
```
Comment (shiroyasha) [inline lib/mix/tasks/ruby.ex:24]: Use http://elixir-lang.org/docs/stable/elixir/IO.ANSI.html for color output.

**Comment by shiroyasha** [comment]
Looks nice 👍 Feel free to merge.

File: lib/mix/tasks/ruby.ex [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  use Mix.Task
+
+  @gen_path "gen/ruby-client"
+  @client_name "test_client"
+
+  @lib_path "#{@gen_path}/lib"
+  @bin_path "#{@gen_path}/bin"
+  @templates_path "templates/ruby"
+
+  @version "0.1.0"
```
Comment (predrag-rakic) [inline lib/mix/tasks/ruby.ex:11]: `mix thrifter.ruby` will be invoked from some server repository (repo in which thrift file lives) - something like [thrifter-demo](https://github.com/renderedtext/thrifter-demo).
`version` and `client_name` should be taken from that (server) repo `mix.file`:
`version = Mix.Project.config[:version]`

Comment (nemanja-m) [reply lib/mix/tasks/ruby.ex:11]: I changed that in https://github.com/renderedtext/thrifter/pull/9.
Please check if something else is missing.

**Pull request merged**